src/Pages/Detail/Detail.js

```diff
@@ -0,0 +1,195 @@
+import { useState, useEffect } from 'react';
+
+import { useHistory } from 'react-router';
+
+import { GET_PRODCUT_API } from '../../config';
+
+import styled from 'styled-components';
```

Comment:
- 사이 간격 안 띄어주시는 게 더 좋아보이는데 어떻게 생각하시나요?
```suggestion
import { useState, useEffect } from 'react';
import { useHistory } from 'react-router';
import { GET_PRODCUT_API } from '../../config';
import styled from 'styled-components';
```

Reply:
넵 ! 적용하였습니다 :)